core/src/main/scala/caliban/wrappers/Wrappers.scala

```diff
@@ -0,0 +1,60 @@
+package caliban.wrappers
+
+import caliban.CalibanError.ExecutionError
+import caliban.Value.NullValue
+import caliban.wrappers.Wrapper.OverallWrapper
+import caliban.{ GraphQL, GraphQLResponse }
+import zio.ZIO
+import zio.clock.Clock
+import zio.console.{ putStrLn, Console }
+import zio.duration.Duration
+
+object Wrappers {
+
+  /**
+   * Attaches to the given GraphQL API definition a wrapper that logs slow queries.
+   * @param duration threshold above which queries are considered slow
+   * @param api a GraphQL API definition
+   */
+  def logSlowQueries[R <: Console with Clock](duration: Duration)(api: GraphQL[R]): GraphQL[R] =
+    api.withWrapper(logSlowQueriesWrapper(duration))
+
+  /**
+   * Returns a wrapper that logs slow queries
+   * @param duration threshold above which queries are considered slow
+   */
+  def logSlowQueriesWrapper(duration: Duration): OverallWrapper[Console with Clock] =
```

Comment:
Nit: Naming should probably be `printSlowQueries`.

Reply:
Good point.